package.json

```diff
@@ -52,7 +53,7 @@
     "openfin-service-async": "^1.0.1",
     "openfin-service-config": "1.0.2",
     "openfin-service-signal": "^1.0.0",
-    "openfin-service-tooling": "1.0.21",
+    "openfin-service-tooling": "1.0.23",
```

Comment:
The latest `openfin-service-tooling` is version `1.0.25`. Can we bring this in as part of this PR (or separate)?